### ISSUE TITLE
Low training accuracy using pre-trained model

### ISSUE THREAD

**Issue by AndreiXYZ** [pytorch/examples#889]
Hello,
I am trying to evaluate a pre-trained mobilenetv2 model from torchvision on the ImageNet training dataset using this script. 
To do so, I modify lines 235-237 to perform validation on the train loader instead of the val loader:
```
    if args.evaluate:
        validate(train_loader, model, criterion, args)
        return
```
Everything else is left untouched. The command I use to run is:
`python imagenet_train_example.py -a mobilenet_v2 -j 16 -b 1024 -e --pretrained /data/ImageNet`
However, the results are lower than expected:
`Acc@1 2.926 Acc@5 15.079 Loss 11.795791`

**Comment by jwli6** [comment]
Have you solved this problem? I had the same situation.

**Comment by mostafaelhoushi** [comment]
Did you try to train for one epoch with learning rate = 0 (`--lr 0`)?
